apistar/compat.py

```diff
@@ -38,8 +38,8 @@ def __repr__(self):
             mode = "" if self.closed else " '%s'" % self.file.mode
             return "<DownloadedFile '%s', %s%s>" % (self.name, state, mode)
 
-            def __str__(self):
-                return self.__repr__()
+        def __str__(self):
+            return self.__repr__()
```

Comment:
Why not `repr(self)`?

Reply:
I just wanted to keep it as it is, so in next PRs I or someone else can do it.